Loader/Config/Settings.lua

```diff
@@ -149,7 +149,7 @@ settings.LocalDatastore = false				 -- If this is turned on, a mock DataStore wi
 settings.Storage = game:GetService("ServerStorage") -- Where things like tools are stored
 settings.RecursiveTools = false					 -- Whether tools that are included in sub-containers within settings.Storage will be available via the :give command (useful if your tools are organized into multiple folders)
 
-settings.Theme = "Default"				-- UI theme;
+settings.Theme = "Modern"				-- UI theme;
```

Comment:
Probably shouldn't make it the default one because there will be people complaining about it like when rounded console was made the default one etc